frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -709,3 +710,52 @@ export const createOverviewItemsForType = (
     return acc;
   }, []);
 };
+
+export const getResourceLimitsData = (limitsData: LimitsData) => ({
```

Comment:
Do we need this function as part of console/shared? I see this only being used in devconsole

Reply:
I am using it in console-app as well